### ISSUE TITLE
Cannot open msteams link to a meeting

### ISSUE THREAD

**Issue by kujaw** [IsmaelMartinez/teams-for-linux#322]
**Describe the bug** **To Reproduce**
I have an msteams link, such as: https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_someMeetingID%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%2522374f8026-7b54-4a3a-b87d-328fa26ec10d%2522%252c%2522Oid%2522%253a%252204031ea7-8917-41d4-8d7f-6dc2340c03d2%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=8870919c-ff9a-4bc7-8f96-52411b010492&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true#

I click on the link in my browser ("Launch it now"), then I click to open it via Microsoft Teams For Linux. Teams launch, but it doesn't open a meeting. If Teams is already running, it focuses on Teams window but also doesn't launch meeting.

**Expected behavior**
Opened meeting in Teams app.

**Desktop (please complete the following information):**
 - OS: Description:	Ubuntu 18.04.3 LTS
 - Installation package: snap
 - Version: 1.0.0, Updated: 05.11.2019

**Comment by IsmaelMartinez** [comment]
Hi, I think it might be related #318

Have you tried the stuff in the comment ? https://github.com/IsmaelMartinez/teams-for-linux/issues/318#issuecomment-584118346

Hope helps

**Comment by IsmaelMartinez** [comment]
Also, snap sometimes does constrain you a bit more. Try deb or rpm if possible. Ta

**Comment by kujaw** [comment]
Yes, I tried one from #318 (comment), but it didn't work. When I click on the link I've sent in a channel, new tab in browser is opened, saying "Your browser doesn't support Teams meetings...."

**Comment by kujaw** [comment]
deb package:
- same behaviour when clicking link inside a channel 
- when trying to open link from web browser by clicking "Launch it now" it looks like it tries to connect several times and in the end I see a message "The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.". Here's log:
```
$ teams-for-linux 
configPath = /home/kujawa/.config/teams-for-linux
Failed to get the config file, using default values
configFile = {}
processArgs [ '/opt/teams-for-linux/teams-for-linux' ]
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=04b26e4d-dd52-41da-81cf-0d8f3c7f2c99&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=04b26e4d-dd52-41da-81cf-0d8f3c7f2c99&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=91696165-ab23-4645-e930-910621cb3651&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=04b26e4d-dd52-41da-81cf-0d8f3c7f2c99&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=91696165-ab23-4645-e930-910621cb3651&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=7eddd85b-6d12-44e0-56ae-2902a4489793&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=04b26e4d-dd52-41da-81cf-0d8f3c7f2c99&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=91696165-ab23-4645-e930-910621cb3651&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=7eddd85b-6d12-44e0-56ae-2902a4489793&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=bad91274-d5b1-4a92-efbf-f2ec1140e7a7&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=e64f876f-9380-4839-e624-2c70a33a1042&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=b2f7d544-3f55-4f00-eadc-28a5b47e5617&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a075752b-7ac3-4799-7e0b-1df8f8fe1049&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=04b26e4d-dd52-41da-81cf-0d8f3c7f2c99&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=91696165-ab23-4645-e930-910621cb3651&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=7eddd85b-6d12-44e0-56ae-2902a4489793&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=bad91274-d5b1-4a92-efbf-f2ec1140e7a7&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=1204765f-ed4f-4311-4bd6-5402f8d1fdc1&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
```

**Comment by IsmaelMartinez** [comment]
have you tried "Ctrl+click" ?

**Comment by kujaw** [comment]
Yes, I tried Ctrl/Alt/Shift + click

**Comment by IsmaelMartinez** [comment]
Can you try following this link and see if you get to the meeting room? 

https://teams.microsoft.com/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com

I am suspecting on this function not doing what it needs todo... 
https://github.com/IsmaelMartinez/teams-for-linux/blob/0984bb160b8e57ab8e51c088637c424a7e989e70/app/mainAppWindow/index.js#L78

**Comment by kujaw** [comment]
Same 
``` 
$ teams-for-linux 
configPath = /home/kujawa/.config/teams-for-linux
Failed to get the config file, using default values
configFile = {}
processArgs [ '/opt/teams-for-linux/teams-for-linux' ]
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a970fa7a-975c-4d3e-eb4b-c82b8fa01ee2&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a970fa7a-975c-4d3e-eb4b-c82b8fa01ee2&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=ddcaa7de-5c32-402f-2050-623296d00add&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a970fa7a-975c-4d3e-eb4b-c82b8fa01ee2&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=ddcaa7de-5c32-402f-2050-623296d00add&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=1cf5d37a-6e83-4ae4-3373-9a34fa1e3fb0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a970fa7a-975c-4d3e-eb4b-c82b8fa01ee2&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=ddcaa7de-5c32-402f-2050-623296d00add&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=1cf5d37a-6e83-4ae4-3373-9a34fa1e3fb0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=2551afef-4ff0-4ef5-30a3-9c09b79e2bfc&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
second-instance started
processArgs [ '/opt/teams-for-linux/teams-for-linux',
  'msteams:/l/meetup-join/19:meeting_M2IxYTc4YjgtNzY1OC00YTU5LWJlYzctMWViZjk0ZjdkM2Jj@thread.v2/0?context=%7b%22Tid%22%3a%22374f8026-7b54-4a3a-b87d-328fa26ec10d%22%2c%22Oid%22%3a%22744eddc4-6b36-4f47-ad6a-eb0d2f572f88%22%7d&anon=true&deeplinkId=23d343ab-9f00-4088-20fb-a623ffbf74b0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true&fqdn=teams.microsoft.com&deeplinkId=aa81039a-177a-4093-0a34-cbad41012899&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=178ddb3d-c0bb-4ebc-8ba9-a71cc17c08e8&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=e16414fc-bb2f-4629-733b-408714004daa&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=a970fa7a-975c-4d3e-eb4b-c82b8fa01ee2&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=ddcaa7de-5c32-402f-2050-623296d00add&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=1cf5d37a-6e83-4ae4-3373-9a34fa1e3fb0&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=2551afef-4ff0-4ef5-30a3-9c09b79e2bfc&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com&deeplinkId=5ed26c2c-fe28-484d-f6f3-daf441705447&launchAgent=join_launcher&type=meetup-join&directDl=true&msLaunch=true&enableMobilePage=true&fqdn=teams.microsoft.com' ]
meetup-join argument received with msteams protocol
```

**Comment by IsmaelMartinez** [comment]
I am looking at #318 and have identified where the issue is but it's a bit of a precision work... Meaning, adding the fix is breaking other functionality... But I am working on it.

**Comment by IsmaelMartinez** [comment]
Pre release ready in here https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.0.4

Can someone test it? I have been running it for 11 days with no issues but there are quite a few changes... internally

**Comment by shayll-herolo** [comment]
If anyone still has issues with this, I fount this comment which fixes my issue on Manjaro 20
https://techcommunity.microsoft.com/t5/microsoft-teams-blog/microsoft-teams-is-now-available-on-linux/bc-p/1064014/highlight/true#M4458
